during_workshop.md

```diff
@@ -212,10 +183,14 @@ The function now expects us to send a JSON object to it containing the subtitle
 
 ![Thunder Client](./images/Thunder-Client-SendRequest.png) 
 
-Take a moment to customise the "httpTrigger" binding a little.
-- We don't want to accept GET requests, so remove that from the list of accepted methods in `function.json`.
-- Rename the binding from "req" to "request". This means updating the "name" property for that binding in `function.json` as well as the corresponding function parameter in `__init__.py`. Each binding in `function.json` needs a corresponding parameter in the `main` function in `__init__.py`, except one output binding can use `main`'s return value instead, if it has name equal to "$return".
+Take a moment to customise the "HttpEndpoint" function a little (by adding parameters to `@app.route`).
+- We only want the function to accept POST requests, so specify this as a `methods` parameter in `@app.route`
+- Rename the argument of the function from `req` to `request` (just changing the name of the argument in the function will **not** work!)

```

Comment:
Very minor, but this feels very much like saying "do X. X will not work" and didn't quite emphasise the "just" in my head
```suggestion
- Rename the argument of the function from `req` to `request` (hint: *just* changing the name of the argument in the function will **not** work!)

```

Reply:
Sounds good to me :)